tflint/tflint.go

```diff
@@ -26,6 +27,17 @@ func RunTflintWithOpts(terragruntOptions *options.TerragruntOptions, terragruntC
 		return err
 	}
 
+	// Get GITHUB_OAUTH_TOKEN and set it as GITHUB_TOKEN so that tflint recognises and respects it
+	githubOauthToken := os.Getenv("GITHUB_OAUTH_TOKEN")
+	if githubOauthToken != "" {
+		err := os.Setenv("GITHUB_TOKEN", githubOauthToken)
+		if err != nil {
+			return errors.WithStackTrace(err)
+		}
+	}
+
+	terragruntOptions.Logger.Debugf("Setting GITHUB_TOKEN to the value of GITHUB_OAUTH_TOKEN")
```

Comment:
This log statement should be in the if-statement above.

Reply:
Oh! Yes, thank you! Here's a new [PR](https://github.com/gruntwork-io/terragrunt/pull/2395) for fixing this.